test/fixtures/config.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
Is this the file where we expect users to define `TF_VAR_*` variables? If so, can we include the variables that users must set + document this file in the README?

Reply:
@adrienthebo I've actually moved in this setup to using `terraform.tfvars` instead of `ENV["TF_VAR_*"]` -- I'll clarify that in the README.

Reply:
Yes, exactly